src/elm/Routes/AboutPage.elm

```diff
@@ -0,0 +1,15 @@
+module Routes.AboutPage exposing (..)
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+import Types exposing (..)
+
+
+aboutPage : Model -> Html Msg
+aboutPage model =
+    div [ class "w-60-ns center" ]
```

Comment:
As you're repeating this line 7 times in different files it's pretty safe to assume this could be its own function. I often like to define something like this:

```elm
layout : Html msg -> Html msg
layout content =
    div [ class "w-60 center" ] [ content ]
```

So you can capture your page container layout.

At this stage I'd also probably recommend that you keep your routes in one file (they're so small at the moment there's no need for them to be in different files) but not essential

Reply:
@andrewMacmurray where would you recommend keeping helper functions like this (assuming we keep them in separate files as node has programmed me this way 😊 )?

Reply:
Sorry don't know why I didn't get this notification. Until you genuinely need to split these files up (like until they're performing very different functionality) I'd keep the helpers in the same file (I usually put them at the bottom with a comment separating them).